test/e2e/apimachinery/flowcontrol.go

```diff
@@ -51,18 +51,18 @@ var _ = SIGDescribe("[Feature:APIPriorityAndFairness]", func() {
 		nonMatchingUsername := "foo"
 
```

Comment:
where is this e2e test running? It has a `Feature:` tag, which excludes it from the default e2e job, and I don't see it opted into in the [alpha e2e job](https://testgrid.k8s.io/google-gce#gci-gce-alpha-enabled-default)

Reply:
shouldn't we remove the feature tag as part of promotion to beta so this is exercised in the default e2e?

Reply:
it's available here: https://testgrid.k8s.io/google-gce#gce-cos-master-alpha-features

Reply:
ah, thanks. question about moving the test to the default e2e job remains.

Reply:
correct, we can expect this test to run through beta :+1: updating